// d3v4/build/d3v4.node.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var d3Array = require('d3-array');
var d3Axis = require('d3-axis');
var d3Collection = require('d3-collection');
var d3Color = require('d3-color');
var d3Dispatch = require('d3-dispatch');
var d3Drag = require('d3-drag');
var d3Ease = require('d3-ease');
var d3Format = require('d3-format');
var d3Hierarchy = require('d3-hierarchy');
var d3Interpolate = require('d3-interpolate');
var d3Queue = require('d3-queue');
var d3Scale = require('d3-scale');
var d3Selection = require('d3-selection');
var d3Shape = require('d3-shape');
var d3Time = require('d3-time');
var d3TimeFormat = require('d3-time-format');
var d3Transition = require('d3-transition');
var d3Voronoi = require('d3-voronoi');
var d3Zoom = require('d3-zoom');

var version = "4.8.0";

exports.version = version;
exports.extent = d3Array.extent;
exports.max = d3Array.max;
exports.merge = d3Array.merge;
exports.min = d3Array.min;
exports.range = d3Array.range;
exports.sum = d3Array.sum;
exports.axisBottom = d3Axis.axisBottom;
exports.axisLeft = d3Axis.axisLeft;
exports.axisRight = d3Axis.axisRight;
exports.axisTop = d3Axis.axisTop;
exports.map = d3Collection.map;
exports.keys = d3Collection.keys;
exports.values = d3Collection.values;
exports.lab = d3Color.lab;
exports.rgb = d3Color.rgb;
exports.dispatch = d3Dispatch.dispatch;
exports.drag = d3Drag.drag;
exports.easeElastic = d3Ease.easeElastic;
exports.easeLinear = d3Ease.easeLinear;
exports.format = d3Format.format;
exports.formatLocale = d3Format.formatLocale;
exports.tree = d3Hierarchy.tree;
exports.interpolate = d3Interpolate.interpolate;
exports.interpolateHsl = d3Interpolate.interpolateHsl;
exports.queue = d3Queue.queue;
exports.scaleBand = d3Scale.scaleBand;
exports.scaleLinear = d3Scale.scaleLinear;
exports.scaleOrdinal = d3Scale.scaleOrdinal;
exports.scaleTime = d3Scale.scaleTime;
exports.schemeCategory20 = d3Scale.schemeCategory20;
exports.arc = d3Shape.arc;
exports.area = d3Shape.area;
exports.curveBasis = d3Shape.curveBasis;
exports.curveCardinal = d3Shape.curveCardinal;
exports.curveLinear = d3Shape.curveLinear;
exports.curveMonotoneX = d3Shape.curveMonotoneX;
exports.curveNatural = d3Shape.curveNatural;
exports.line = d3Shape.line;
exports.pie = d3Shape.pie;
exports.stack = d3Shape.stack;
exports.stackOffsetExpand = d3Shape.stackOffsetExpand;
exports.stackOffsetNone = d3Shape.stackOffsetNone;
exports.stackOffsetSilhouette = d3Shape.stackOffsetSilhouette;
exports.stackOffsetWiggle = d3Shape.stackOffsetWiggle;
exports.stackOrderInsideOut = d3Shape.stackOrderInsideOut;
exports.stackOrderNone = d3Shape.stackOrderNone;
exports.symbol = d3Shape.symbol;
exports.timeDay = d3Time.timeDay;
exports.timeMonth = d3Time.timeMonth;
exports.timeMonths = d3Time.timeMonths;
exports.timeFormat = d3TimeFormat.timeFormat;
exports.timeFormatLocale = d3TimeFormat.timeFormatLocale;
exports.timeParse = d3TimeFormat.timeParse;
exports.transition = d3Transition.transition;
exports.voronoi = d3Voronoi.voronoi;
exports.zoom = d3Zoom.zoom;
exports.zoomIdentity = d3Zoom.zoomIdentity;
Object.keys(d3Selection).forEach(function (key) { exports[key] = d3Selection[key]; });
Object.defineProperty(exports, "event", {get: function() { return d3Selection.event; }});
