// d3v4/build/package.js
export var _args = [[{"raw":"d3@4.8.0","scope":null,"escapedName":"  ","name":"d3","rawSpec":"4.8.0","spec":"4.8.0","type":"version"},"/Users/hrogers/Sites/sucrose"]];
export var _from = "d3@4.8.0";
export var _id = "d3@4.8.0";
export var _inCache = true;
export var _location = "/d3";
export var _nodeVersion = "7.3.0";
export var _npmOperationalInternal = {"host":"packages-12-west.internal.npmjs.com","tmp":"tmp/d3-4.8.0.tgz_1492234567531_0.6074790894053876"};
export var _npmUser = {"name":"mbostock","email":"mike@ocks.org"};
export var _npmVersion = "3.10.10";
export var _phantomChildren = {};
export var _requested = {"raw":"d3@4.8.0","scope":null,"escapedName":"d3","name":"d3","rawSpec":"4.8.0","spec":"4.8.0","type":"version"};
export var _requiredBy = ["#USER","/"];
export var _resolved = "https://cache.sugardev.team/repository/npm/d3/-/d3-4.8.0.tgz";
export var _shasum = "1ad8d18997869c90b6ad6114e9b92425cee78460";
export var _shrinkwrap = null;
export var _spec = "d3@4.8.0";
export var _where = "/Users/hrogers/Sites/sucrose";
export var author = {"name":"Mike Bostock","url":"https://bost.ocks.org/mike"};
export var browser = "build/d3.js";
export var bugs = {"url":"https://github.com/d3/d3/issues"};
export var dependencies = {"d3-array":"1.2.0","d3-axis":"1.0.6","d3-brush":"1.0.4","d3-chord":"1.0.4","d3-collection":"1.0.3","d3-color":"1.0.3","d3-dispatch":"1.0.3","d3-drag":"1.0.4","d3-dsv":"1.0.5","d3-ease":"1.0.3","d3-force":"1.0.6","d3-format":"1.2.0","d3-geo":"1.6.3","d3-hierarchy":"1.1.4","d3-interpolate":"1.1.4","d3-path":"1.0.5","d3-polygon":"1.0.3","d3-quadtree":"1.0.3","d3-queue":"3.0.5","d3-random":"1.0.3","d3-request":"1.0.5","d3-scale":"1.0.5","d3-selection":"1.0.5","d3-shape":"1.0.6","d3-time":"1.0.6","d3-time-format":"2.0.5","d3-timer":"1.0.5","d3-transition":"1.0.4","d3-voronoi":"1.1.2","d3-zoom":"1.1.4"};
export var description = "Data-Driven Documents";
export var devDependencies = {"json2module":"0.0","package-preamble":"0.0","rimraf":"2","rollup":"^0.41.4","rollup-plugin-ascii":"0.0","rollup-plugin-node-resolve":"3","tape":"4","uglify-js":"^2.8.11"};
export var directories = {};
export var dist = {"shasum":"1ad8d18997869c90b6ad6114e9b92425cee78460","tarball":"https://cache.sugardev.team/repository/npm/d3/-/d3-4.8.0.tgz"};
export var gitHead = "02d5392d825930b2bd27e87d1d9bc44a639e10f6";
export var homepage = "https://d3js.org";
export var keywords = ["dom","visualization","svg","animation","canvas"];
export var license = "BSD-3-Clause";
export var main = "build/d3v4.node.js";
export var maintainers = [{"name":"mbostock","email":"mbostock@gmail.com"},{"name":"jasondavies","email":"jason@jasondavies.com"}];
export var module = "index";
export var name = "d3v4";
export var optionalDependencies = {};
export var readme = "ERROR: No README data found!";
export var repository = {"type":"git","url":"git+https://github.com/d3/d3.git"};
export var scripts = {"build":"rollup -c --banner \"$(preamble)\" -f umd -n d3v4 -o build/d3v4.js -- index.js && uglifyjs --preamble \"$(preamble)\" build/d3v4.js -c negate_iife=false -m -o build/d3v4.min.js","pretest":"rimraf build && mkdir build && json2module package.json > build/package.js && node rollup.node","test":"tape 'test/**/*-test.js'"};
export var version = "4.8.0";
